src/frontends/paddle/src/op/tile.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+#include "openvino/opsets/opset6.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs tile(const NodeContext& node) {
+    auto x = node.get_input("X");
+    Output<Node> repeats;
+    if (node.has_input("RepeatTimes")) {
+        repeats = node.get_input("RepeatTimes");
+    } else {
+        std::vector<int32_t> repeats_vector = node.get_attribute<std::vector<int32_t>>("repeat_times", {});
+        repeats = ov::opset6::Constant::create(ov::element::i32, Shape{repeats_vector.size()}, repeats_vector);
```

Comment:
please use the default_opset , not ov::opset6

Reply:
done